executor/prepared_test.go

```diff
@@ -70,8 +70,28 @@ func (s *testSuite) TestPrepared(c *C) {
 		query := "select c1, c2 from prepare_test where c1 = ?"
 		stmtId, _, _, err := tk.Se.PrepareStmt(query)
 		c.Assert(err, IsNil)
-		_, err = tk.Se.ExecutePreparedStmt(goCtx, stmtId, 1)
+		rs, err := tk.Se.ExecutePreparedStmt(goCtx, stmtId, 1)
```

Comment:
Does it help?
`tk.MustQuery("select c1, c2 from prepare_test where c1 = ?", 1)`

It's basically the same with your prepared statement.

Reply:
I just add a result check for this. I think these code are used to call prepare and execute directly.